### ISSUE TITLE
Need to represent RCV Results

### ISSUE THREAD

**Issue by carl3** [usnistgov/ElectionResultsReporting#4]
**Organization Name**: Carl Hage

**Organization Type**: 4

**Document**: SP1500-100 ElectionResultsReporting

**Reference**: 4.2.8 Counts

**Comment**: There is a way to define RCV as a vote variation, but no defined way to represent the result totals. With RCV, a set of elimination rounds occurs, moving votes from eliminated candidates to the next choices for each ballot, until a winner is obtained (e.g. has a majority of votes for a single seat contest). [Candidates are eliminated one-by-one like a reality TV show, but TV shows don't allow viewers to rank choices, even though dance judges do!]

**Suggested Change**: Add `<RCVRound>` an integer element to the Contest element, an additional filter qualifying the results reported. The round would be an integer 1 to n, where the highest round has final results.

An alternative [better] interpretation is that `<RCVRound>` is used to represent rounds prior to the final round-- an unqualified set of totals is used to represent the final round (no `<RCVRound>` present). In this case, the presence of `<RCVRound>` means results are a non-final subtotal. It may be better to use the second interpretation, since it is common to have results posted just for RCV round 1 on election night-- the full rounds are processed and reported later. Adding `<RCVRound>1</RCVRound>` indicates the totals are not final. Higher numbers show subtotals prior to the final round with complete vote tally.

**Comment by jdmgoogle** [comment]
There also needs to be something added to the SummaryCounts to indicate the number of ballots which have been exhausted at each round.

**Comment by carl3** [comment]
Yes, FYI exausted ballots mean none of a voters choices won and they didn't get a chance to rank between the final 2. If you had 10 candidates and only 3 choices to rank, your vote might get discarded. Exausted ballot counts let people know when the number of choices available for ranking is inadequate.

**Comment by johnpwack** [comment]
We are adding a round indication as well as number of ballots exhausted to the 2nd version of the spec, as per Justin.
